gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/pages/PermissionsPage.java

```diff
@@ -58,18 +59,21 @@ public PermissionsPage() {
 		disableLink(this.permissionsPageLink);
 	}
 
-	public PermissionsPage(final GbUser taSelected) {
-		disableLink(this.permissionsPageLink);
-		this.taSelected = taSelected;
-	}
-
 	@Override
 	public void onInitialize() {
 		super.onInitialize();
 
+		// grab the selected parameter (if provided)
+		final PageParameters params = getPageParameters();
+		final Optional<String> taUuid = Optional.ofNullable(params.get("selected").toOptionalString());
```

Comment:
If its clearer that its just a string that might have a value or not then maybe this should just be a string, likewise `taSelected`. Might be a bit overkill to wrap it all in an Optional when we can just `StringUtils.IsBlank` it?